frontend/src/static/js/v_summary.js

```diff
@@ -267,6 +267,10 @@ window.vSummary = {
       return res;
     },
 
+    getAuthorProfileLink(userName) {
+      return `https://github.com/${userName}`;
```

Comment:
I see that we are using `https://github.com` at several points in our js files.
Perhaps we could keep the base url as a variable. This could be done in a separate PR, if it's appropriate.

Reply:
It can be done in this PR actually since it is not a big change and we are already touching this. However, it is also possible to be a `first timer` issue to be solved.

Reply:
Better to keep it as a `firstTimer` issue. Will open a new issue